### DISCUSSION TITLE
Nuestra Mejor Semana

### DISCUSSION THREAD

**Discussion by TamaraPiccinni** [CodeSystem2022/Binary-Brains_Tercer-Semestre#122] [Announcements]
Proyecto semana 10 fue la seleccionada para ser la mejor!!! en la que aplicaremos todos nuestros conocimientos y usaremos para presentar.

**Comment by NataliaLiscio1985** [comment]
Trabajamos muy bien, colaboramos todos los miembros del equipo.

**Reply by LuciSere** [reply]
Muy buena coordinación entre todos.

**Comment by mauriciosandrea** [comment]
lo mejor en todo y exelente ambiente de trabajo........
